tests/integration_tests/data_steward/cdr_cleaner/cleaning_rules/create_person_ext_table_test.py

```diff
@@ -0,0 +1,186 @@
+"""
+Original Issues: DC-1012, DC-1514
+
+Background
+In order to avoid further changes to the standard OMOP person table, five non-standard fields will be housed in a
+person_ext table.
+
+Cleaning rule script to run AFTER deid. This needs to happen in deid_base. It depends on the cleaning rules applied in 
+deid to be correctly de-identified.
+This cleaning rule will populate the person_ext table
+The following fields will need to be copied from the observation table:
+src_id (from observation_ext, should all be “PPI/PM”)
+state_of_residence_concept_id: the value_source_concept_id field in the OBSERVATION table row where
+observation_source_concept_id  = 1585249 (StreetAddress_PIIState)
+state_of_residence_source_value: the concept_name from the concept table for the state_of_residence_concept_id
+person_id (as research_id) can be pulled from the person table
+sex_at_birth_concept_id: value_as_concept_id in observation where observation_source_concept_id = 1585845
+sex_at_birth_source_concept_id: value_source_concept_id in observation where observation_source_concept_id = 1585845
+sex_at_birth_source_value: concept_code in the concept table where joining from observation where 
+observation_source_concept_id = 1585845
+"""
+
+# Python imports
+import os
+
+# Project imports
+from app_identity import PROJECT_ID
+from cdr_cleaner.cleaning_rules.create_person_ext_table import CreatePersonExtTable
+from tests.integration_tests.data_steward.cdr_cleaner.cleaning_rules.bigquery_tests_base import BaseTest
+
+
+class CreatePersonExtTableTest(BaseTest.CleaningRulesTestBase):
+
+    @classmethod
+    def setUpClass(cls):
+        print('**************************************************************')
+        print(cls.__name__)
+        print('**************************************************************')
+
+        super().initialize_class_vars()
+
+        # set the test project identifier
+        project_id = os.environ.get(PROJECT_ID)
```

Comment:
if the import changes, this should use:

```suggestion
        project_id = get_application_id()
```